sdk/keyvault/azure-security-keyvault-secrets/src/secret_serializers.cpp

```diff
@@ -152,3 +152,38 @@ void KeyVaultDeletedSecretSerializer::KeyVaultDeletedSecretDeserialize(
   secret.DeletedDate
       = PosixTimeConverter::PosixTimeToDateTime(jsonParser[_detail::DeletedDatePropertyName]);
 }
+
+std::string KeyVaultSecretPropertiesSerializer::KeyVaultSecretPropertiesSerialize(
+    KeyvaultSecretProperties const& properties)
+{
+  Azure::Core::Json::_internal::json payload;
```

Comment:
nit. 
add `using namespace Azure::Core::Json::_internal` to the top of the file so this line and all other can be shortened as `json payload` only